docs/en/setup/backend/backend-storage.md

```diff
@@ -103,12 +104,24 @@ storage:
     bulkSize: ${SW_STORAGE_ES_BULK_SIZE:20} # flush the bulk every 20mb
     flushInterval: ${SW_STORAGE_ES_FLUSH_INTERVAL:10} # flush the bulk every 10 seconds whatever the number of requests
     concurrentRequests: ${SW_STORAGE_ES_CONCURRENT_REQUESTS:2} # the number of concurrent requests
+    advanced: ${SW_STORAGE_ES_ADVANCED:""}
 ```
 
 
 ### Data TTL
 TTL in ElasticSearch overrides the settings of core, read [ElasticSearch section in TTL document](ttl.md#elasticsearch-6-storage-ttl)
 
+### Advanced Configurations For Elasticsearch Index
+You can add advanced configurations in `JSON` format to set `elasticsearch index settings`.
+For example, set [translog](https://www.elastic.co/guide/en/elasticsearch/reference/master/index-modules-translog.html) settings:
```

Comment:
I think this doc ref will misguide people. From my understanding, all parameters in [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html) should be supported, right?

Reply:
Yes.

Reply:
Need to add this ref https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html ?

Reply:
Then you should ref to the config page in the document, your example could be same. Such as
___
You can add advanced configurations in `JSON` format to set `ElasticSearch index settings` by following [ElasticSearch doc](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html)

Reply:
Thanks, I will add it.